Source/WTF/wtf/UUID.cpp

```diff
@@ -126,7 +126,12 @@ std::optional<UUID> UUID::parse(StringView value)
     if (result == deletedValue)
         return { };
 
-    return UUID(result);
+    return result;
+}
+
+std::optional<UUID> UUID::parse(StringView value)
+{
+    return UUID(convertStringToInt(value));
 }
```

Comment:
This change doesn't seem right to me. The parse() function now unconditionally returns a UUID. That means the interface advertises an optional return value but never returns nullopt, and there's no explicit way to signal failure to parse.

I'd suggest retaining the old interface / behavior.